src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -86,6 +86,8 @@
     </Compile>
     <!-- nuget.exe localized assemblies -->
     <EmbeddedResource Include="$(OutputPath)\**\$(AssemblyName).resources.dll" Exclude="@(MergeExclude)" />
+    <!-- NuGet.Commands localized assemblies -->
+    <LocResource Include="$(OutputPath)\**\NuGet.Commands.resources.dll" Exclude="@(MergeExclude)" />
```

Comment:
I'm just curious: what is the difference between this one and `string resourceName = $"NuGet.CommandLine.{culture.Name}.{name}.dll";` in line 269? Which one contains exactly which resources?

Reply:
Below is a comparison table.

Name | Description | How it's shipped in `nuget.exe`
----|-----|----
`Nuget.CommandLine.{culture.Name}.{name}.dll` | This is the name of a DLL embedded as a resource. Since we rename NuGet.CommandLine.exe to NuGet.exe, {name} part becomes "NuGet.Resources" | Embedded in NuGet.exe
`NuGet.Commands.resources.dll` | Contains all strings from NuGet.Commands project | ILMerged

LMK if you have more questions.